packages/core/src/Stores/client-store.js

```diff
@@ -842,11 +841,6 @@ export default class ClientStore extends BaseStore {
         ).length;
     }
 
-    get should_show_eu_content() {
-        const is_current_mf = this.landing_company_shortcode === 'maltainvest';
-        return (!this.is_logged_in && this.is_eu_country) || this.is_eu || is_current_mf;
-    }
-
     get should_show_eu_error() {
```

Comment:
@maryia-deriv should we remove this as well?

Reply:
if you mean `should_show_eu_error` - no, it's added and now used by Bots.